trashtalk/views/cleanups.py

```diff
@@ -177,25 +176,17 @@ def create():
     :return: redirect to cleanup/:id if successful, to cleanup/new if not
     """
     # TODO: Issue #9 - Prevent multiple accidental submissions (ex., when errors occur)
-    full_address= get_full_address(request.form.to_dict())
+    full_address = get_full_address(request.form.to_dict())
     try:
         geoloc = get_location(full_address)
-    except:
+    except:  # TODO: Issue 56 - Add custom exception CleanupException or something
         current_app.logger.exception("There was an error finding location.")
-        flash("s%: Address not found" % full_address)
+        flash("There was an error creating your cleanup. Please try again.")
         return redirect(url_for('cleanups.new'))
     else:
-        location=location_factory(geoloc._asdict())
+        location = location_factory(geoloc._asdict())
         cleanup_data = request.form.copy()
-        if not cleanup_data['image']:
-            current_app.logger.info("No image was found: replaced with default")
-            cleanup_data['image'] = "crossed_rakes.png"
-        else:
-            current_app.logger.info("Image found: ", cleanup_data['image'])
-        cleanup_data['start_time'] = request.form["start_time"]
-        cleanup_data['end_time'] = request.form["end_time"]
```

Comment:
These two lines will serve a purpose: They swap *12hr am/pm* inputs to *24hr* SQL friendly format using a function from **input_handling.py**

Reply:
These lines only created a new object to start the process of saving the cleanup to the database. They did not do time conversion. 

Even if they did, Python does the time conversion for us via Sqlalchemy.